src/checkers/inference/model/serialization/ToStringSerializer.java

```diff
@@ -24,49 +28,65 @@
 import checkers.inference.model.VariableSlot;
 
 /**
- * This Serializer is meant only to convert constraints and variables to
- * human readable strings.  It is used currently by the DebugSolver.
+ * This Serializer is meant only to convert constraints and variables to human readable strings. It
+ * is used currently by the DebugSolver.
  */
 public class ToStringSerializer implements Serializer<String, String> {
     private final boolean showAstPaths;
     private boolean showVerboseVars;
 
-    private int indent = 0;
-    public static final String INDENT_STRING = "    ";
+    private int indentationLevel = 0;
+
+    // 2 spaces per indentation
+    public static final String INDENT = "  ";
+
+    // stores N concatenations of INDENT at index N, where index 0 stores empty string
```

Comment:
Maybe use Javadoc and use {@link #INDENT} and {@link indentationLevel} is much clear.